docs/developer/community/contributing.mdx

```diff
@@ -265,6 +265,7 @@ Use relative imports.
 ### Migrations
 
 Try to combine multiple migrations into one, but remember not to mix changes on the database with updating rows in migrations. In other words, operations that alter tables and use `RunPython` to run methods on existing data should be in separate files.
+From Saleor version 3.13 forward follow [zero-downtime policy](zero-downtime-migrations) when writing migrations.
```

Comment:
I think we should link to files so the link gets validated. Relative paths are generated automatically during the build phase.